api/predict.py

```diff
@@ -30,7 +30,7 @@
 
 # Creating a JSON response model: https://flask-restplus.readthedocs.io/en/stable/marshalling.html#the-api-model-factory
 label_prediction = MAX_API.model('LabelPrediction',
-                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})
+                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})  # noqa
```

Comment:
What is the issue of this line?

Reply:
I got it, it's fine.

```suggestion
                                 {l: fields.Float(required=True, description='Class probability') for l in class_labels})  # noqa: E741
```

Reply:
The newest version of `flake8` returns this error, older versions do not:

```
./api/predict.py:33:103: E741 ambiguous variable name 'l'
```